index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
Your HTML code are "okay". I know at the beginning you will forget some tags and semantic. For example the "meta" tag it's very important for the web browsers to understand better your code, for the RP and for the search engines.